### PR TITLE
add --persistnoncorr

### PR DESCRIPTION
**Pull request by wonderingabout**
persist only for non-correspondence games

### CODE REVIEW

**Comment by Dorus** [comment]
<3 this one. Will test it soon.

**Comment by Dorus** [comment]
Code looks good, but can you add a test or two?

**Comment by wonderingabout** [comment]
sure, i just need to learn/find/guess how to do a test in game.js :sweat_smile: 
i think it's something i can do, this evening i'll give it a try

**Comment by Dorus** [comment]
You only really need to call `getBotMoves` in the test. If you cannot find it, i'll write one.

**Comment by wonderingabout** [comment]
ah i see ty :)
i want to give it a few tries and write it myself if possible :smiley:

File: test/test.js [hunk 1]
```diff
@@ -797,24 +797,71 @@ describe("Pv should work", () => {
...
         let conn = new connection.Connection(() => { return fake_socket; }, config);
+
         const game = sinon.spy();
         game.sendChat = sinon.spy();
         game.processing = true;
         game.state = { width: 19, moves: { length: 2 } };
+
         config.ogspv = pvCode;
+
         new Bot(conn, game, config.bot_command);
```
Comment (wonderingabout) [inline test/test.js:818]: independently from the persist PR, is something missing in this line?
ex: https://github.com/online-go/gtp2ogs/blob/devel/test/test.js#L757

**Comment by wonderingabout** [comment]
so i still dont have enough knowledge to write the tests for game.js, but glad i gave it a few tries, it helped me understand things :1st_place_medal: 

i rebased in 1 commit all changes except for test.js, could you PR me the tests for persist and persistnoncorr @Dorus ?

**Comment by Dorus** [comment]
This can be merged once travis is green.

**Pull request merged**